### DISCUSSION TITLE
Better organization topics support

### DISCUSSION THREAD

**Discussion by destumme** [community/community#44672] [Repositories]
### Select Topic Area

Product Feedback

### Body

The topics feature is great for adding quick tags to repos, but I feel there should be a landing page for listing/searching topics just within an organization. Doing a generic search with `org:<org> topic:<topic>` seems to work fine, but when a topic is selected from an internal/org owned repo it redirects to a public search page: `https://github.com/topics/<topic>`. There is no `https://github.com/<org>/topics/<topic>`, which would make organizing repos within large orgs much more straightforward.

**Comment by merusso** [comment]
This would be very useful for organizations! Being unable to list topics in an org limits the usefulness for topics, since they aren't discoverable.
